docs/gadget-devel/gadget-wasm-api-go.md

```diff
@@ -0,0 +1,9 @@
+---
+title: 'wasm-go'
+weight: 110
+description: 'Wasm Golang API'
+---
+
+TODO: how to point to the right version
```

Comment:
What about just removing the version?

https://pkg.go.dev/github.com/inspektor-gadget/inspektor-gadget/wasmapi/go